include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -783,13 +783,13 @@ class TwoTargetOp<string mnemonic, list<Trait> traits = []> :
                 traits)>;
 
 //===----------------------------------------------------------------------===//
-// Quantum operators (gates)
+// Quantum operations (gates)
```

Comment:
These were called "operators" because an operation is already an MLIR term of art. Using the same noun for both could lead to confusion.

On the other hand, it is unclear to me why we didn't just call these "gates". I suppose some of the operators may pedantically not be "true quantum gates" or something like that?

Reply:
I just aligned the comment in the code with what the documentation calls them; we have a section on quantum operators in the docs (which doesn't cover these), and we have a section on quantum operations in the docs, which documents this. Also, the title "Quantum operators (gates)" occurs twice in this file. But I see what you mean regarding MLIR terminology. Here is where quantum computing terminology and compiler terminology clash, I guess. Gates is on the other hand usually causing confusion for people from EE... I'll update this to unitary quantum transformation, and leave it up to the next person if you want to name it differently.